README.md

```diff
@@ -117,6 +117,62 @@ cron.schedule('* * * Jan,Sep Sun', function(){
 });
 ```
 
+## Cron methods
+
+### Schedule
+
+Schedules given task to be executed whenever the cron expression ticks.
+
+Arguments:
+
+- !string expression - Cron expression
+- !Function func - Task to be executed
+- boolean? immediateStart - Whether to start scheduler immediately after create.
```

Comment:
would it be better to create a method `executeAndSchedule` (or two "objects", or something else) instead of passing this flag?